sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -202,7 +203,7 @@ def _credentials_override(f):
     If 'base64_password' is not set, it will fallback to to 'password' in config.
     """
     credentials = f()
```

Comment:
This is where you could see if auth_type is there to begin with or not. Please also test it with an actual file.